### DISCUSSION TITLE
debug_trace not working on full node

### DISCUSSION THREAD

**Discussion by anonymous-sushi** [ethereum-optimism/developers#285] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I am trying to debug_trace fresh transactions coming from new blocks and i am getting
`{'jsonrpc': '2.0', 'id': 1, 'error': {'code': -32601, 'message': 'the method debug_traceTransaction does not exist/is not available'}}`

I have op-geth and op-node running with these params

/root/op-geth/build/bin/geth --http --http.port=8545 --http.addr=localhost --ws --authrpc.addr=localhost --authrpc.jwtsecret=/root/op-geth/jwt.txt --verbosity=3 --rollup.sequencerhttp=https://mainnet-sequencer.optimism.io/ --op-network=op-mainnet --datadir=/root/op_data --http.vhosts=*

/root/optimism/op-node/bin/op-node --l1=http://xxxxx/rpc --l1.rpckind=standard --l1.beacon=http://xxxxx/beacon --l2=ws://localhost:8551 --l2.jwt-secret=/root/optimism/op-node/jwt.txt --network=op-mainnet --syncmode=execution-layer --rpc.port=10545

according to the docs this should work?

### Protocol Description

op-geth: latest
op-node:  latest
network: optimism
chainId: optimism chain
L1 chain: eth

**Comment by imtipi** [comment]
try something like --http.api=web3,debug,eth,txpool,net,engine --ws.api=debug,eth,txpool,net,engine,web3 in the opgeth side

**Reply by anonymous-sushi** [reply]
thanks, wish this was in the docs!
